src/Microsoft.OData.Core/ODataReaderCore.cs

```diff
@@ -282,6 +300,17 @@ protected bool ReadingResourceSet
         }
 
         /// <summary>
+        /// Set to true if a delta response is being read.
+        /// </summary>
+        protected bool ReadingDelta
+        {
+            get
+            {
+                return this.readingDelta;
+            }
+        }
+
+        /// <summary>
         /// Returns true if we are reading a nested payload,
         /// e.g. an expanded resource or resource set within a delta payload,

```

Comment:
>rce or resource set within a delta payload, [](start = 34, length = 43)

API update?

Reply:
Actually, this still returns true when reading a resource set within a delta payload, it's just that now that it's integrated there will always be a listener whether it's an expanded resource or resource set within a delta payload.

---
In reply to: [150161667](https://github.com/OData/odata.net/pull/989#discussion_r150161667) [](ancestors = 150161667)